src/app/views/teams/teams-view/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { connect } from "react-redux";
+import { loadTeamsRequest } from "../../../config/thunks";
+import TeamsList from "./TeamsList";
+
+function mapStateToProps(state) {
+    return {
+        teams: state.state.teams.all,
```

Comment:
you can use https://github.com/ONSdigital/florence/blob/develop/src/app/config/selectors.js#L29